package.json

```
@@ -75,11 +75,9 @@
   },
   "dependencies": {
     "@shopify/network": "^3.2.1",
-    "@types/node-fetch": "^2.5.7",
-    "@types/supertest": "^2.0.10",
+    "compare-versions": "^6.0.0-rc.1",
```

Prefer to use latest stable release rather than a release candidate ... we can update it later once `6.0.0` has been released.
```suggestion
    "compare-versions": "^5.0.3",
```